assets/src/models/createDetourMachine.ts

```diff
@@ -348,51 +358,35 @@ export const createDetourMachine = setup({
               },
             },
             "Finished Drawing": {
-              invoke: [
-                {
-                  src: "fetch-finished-detour",
-                  input: ({
-                    context: { routePattern, startPoint, endPoint },
-                  }) => ({
-                    routePatternId: routePattern?.id,
-                    startPoint,
-                    endPoint,
-                  }),
-
-                  onDone: {
-                    actions: assign({
-                      finishedDetour: ({ event }) => event.output,
-                    }),
-                  },
-
-                  onError: {},
-                },
-
-                {
-                  src: "fetch-detour-directions",
-                  input: ({
-                    context: { startPoint, waypoints, endPoint },
-                  }) => ({
-                    points: (startPoint ? [startPoint] : [])
-                      .concat(waypoints || [])
-                      .concat(endPoint ? [endPoint] : []),
-                  }),
+              invoke: {
+                src: "fetch-finished-detour",
```

Comment:
NGL - it was pretty neat that I was able to do what I was trying to do by just going here and removing one of the `invoke`s.